nixos/modules/services/web-apps/mobilizon.nix

```diff
@@ -284,9 +286,9 @@ in
             install -m 600 /dev/null "${secretEnvFile}"
             cat > "${secretEnvFile}" <<EOF
           # This file was automatically generated by mobilizon-setup-secrets.service
-          export MOBILIZON_AUTH_SECRET='$(${evalElixir} "${genSecret}")'
-          export MOBILIZON_INSTANCE_SECRET='$(${evalElixir} "${genSecret}")'
-          export RELEASE_COOKIE='$(${evalElixir} "${genCookie}")'
+          export MOBILIZON_AUTH_SECRET='$(${evalElixir genSecret})'
```

Comment:
A comment if you are interested.
Secrets should not be read from environment variables at the risk of leaking to other processes or in the logs.
For plausible, there was a PR submitted upstream that enables to read secrets from files or from an environment variable (to support legacy behavior). That means that it doesn't break existing users but enables secrets to be read from files (which is the recommended way to do it). https://github.com/plausible/analytics/pull/1129
(after that passing secrets is recommended to do through LoadCredentials. If you look into the plausible service, it's all there.
I understand that it's quite some work to submit PR upstream, so feel free to ignore this if you don't have time.

Reply:
Interesting, didn't know about that. After getting nothing done about it for months I just decided that I won't work on a PR for upstream, just don't have enough time for it atm...